src/components/utils/charcount/charcount.component.tsx

```diff
@@ -0,0 +1,14 @@
+import clsx from 'clsx';
+import React from 'react';
+
+export interface ICharCountProps {
+  value: string;
+}
+
+/**
+ * Reusable component showing `count` as number of characters typed into the component.
+ */
+export const CharCount: React.FC<ICharCountProps> = ({value}) => {
+  const className = clsx(`of-charcount`);
+  return <span className={className}>{value.length} characters</span>;
```

Comment:
one next step after this is merged is to properly localize this with react-intl, because now the pluralization is not handled properly

Reply:
Correct